File: POO_Semana1/Pytho_Semana1/Ejercicio06/Loan-books.cpp
#include <iostream>
using namespace std;

class Book{
    public:
    string title;
    string author;
    int year;
    bool available;

    string Borrow(){
        if(available){
            available = false;
            return "Prestamo exitoso.";
        }else{
            return "Error libro no disponible";
        }

    }
    
    string BringBack(){
        available = true;
        return "devolucion exitosa";
    }
};

int main(){
    Book book1;
    book1.title = "To Kill a Mockingbird";
    book1.author = "harper Lee";
    book1.year = 1960;
    book1.available = true;

    string prestamo1 = book1.Borrow();
    string devolucion1 = book1.BringBack();
    
    cout << prestamo1 << endl;
    cout << devolucion1 << endl;
}

File: POO_Semana1/Pytho_Semana1/Ejercicio06/Loan-Books.py
class Book:
    title = ""
    author = ""
    year = 0
    available = True

    def Borrow(self):
        if self.available:
            self.available=False
            return f"Prestamo exitoso"
        else:
            return f"El libro: {self.title}, no esta disponible" 
    def BringBack(self):
        self.available=True
        return f"Libro devuelto"
book1 = Book()
book1.title ="To Kill a Mockingbird"
book1.author = "Harper Lee"
book1.year = 1960
book1.available = True

prestamo1 = book1.Borrow()
devolucion1 = book1.BringBack()
print(prestamo1)
print(devolucion1)
    